rabbitmq/requirements-dev.txt

```diff
@@ -1 +1,2 @@
 datadog-checks-dev
+pika
```

Comment:
should we pin this one?

Reply:
pinning it to what it was in the old test deps